Robinhood/Robinhood.py

```diff
@@ -1302,8 +1201,144 @@ def submit_order(self,
+                ('instrument', instrument_URL),
+                ('symbol', symbol),
+                ('type', order_type),
+                ('time_in_force', time_in_force),
+                ('trigger', trigger),
+                ('price', price),
+                ('stop_price', stop_price),
+                ('quantity', quantity),
+                ('side', side)
+            ]:
+            if(value is not None):
+                payload[field] = value
+
+        print(payload)
+
+        res = self.session.post(endpoints.orders(), data=payload, timeout=15)
+        res.raise_for_status()
+
+        return res
+    
+    def submit_buy_order(self,
+                     instrument_URL=None,
+                     symbol=None,
+                     order_type=None,
+                     time_in_force=None,
+                     trigger=None,
+                     price=None,
+                     stop_price=None,
+                     quantity=None,
+                     side=None):
+        """Submits buy order to Robinhood
+       
+
+            Notes:
+                This is normally not called directly.  Most programs should use
+                one of the following instead:
+
+                    place_market_buy_order()
+                    place_limit_buy_order()
+                    place_stop_loss_buy_order()
+                    place_stop_limit_buy_order()
+                    place_market_sell_order()
+                    place_limit_sell_order()
+                    place_stop_loss_sell_order()
+                    place_stop_limit_sell_order()
+
+            Args:
+                instrument_URL (str): the RH URL for the instrument
+                symbol (str): the ticker symbol for the instrument
+                order_type (str): 'MARKET' or 'LIMIT'
+                time_in_force (:enum:`TIME_IN_FORCE`): GFD or GTC (day or
+                                                       until cancelled)
+                trigger (str): IMMEDIATE or STOP enum
+                price (float): The share price you'll accept
+                stop_price (float): The price at which the order becomes a
+                                    market or limit order
+                quantity (int): The number of shares to buy/sell
+                side (str): BUY or sell
+
+            Returns:
+                (:obj:`requests.request`): result from `orders` put command
+        """
+
+        # Used for default price input
+        # Price is required, so we use the current ask price if it is not specified
+        current_quote = self.get_quote(symbol)
+        current_ask_price = current_quote['ask_price']
+
+        # Start with some parameter checks. I'm paranoid about $.
+        if(instrument_URL is None):
+            if(symbol is None):
+                raise(ValueError('Neither instrument_URL nor symbol were passed to submit_order'))
+            instrument_URL = self.instruments(symbol)[0]['url']
```

Comment:
This was exactly the cause of #76

Reply:
As per @jasonleehodges 's suggestion in #76:

```suggestion
            instrument_URL = self.get_url(self.quote_data(stock)["instrument"])["id"]
```